SrcDragonfly/DRAGONFLY_Parallel_Start.m

```diff
@@ -122,16 +121,25 @@
 %%% _core FUNCTION GENERATION %%%
 
 % Retrive information about the function caller.
+
+% The dbstack has to be reviewed, I don't know why but IK have to do it
 fCallerInformation=evalin('caller', 'dbstack(1)');
-fName=fCallerInformation.file;
-fPath=[evalin('caller', 'pwd') '\'];
-fPathAndName=[evalin('caller', 'pwd') '\' fName];
+if (strcmp(Parallel.OperatingSystem, 'unix'))
```

Comment:
this can be made more compact using filesep?
for example
`fPathAndName=[evalin('caller', 'pwd') filesep fName];` 

should work for both unix and win, instead of the two separate lines
`fPathAndName=[evalin('caller', 'pwd') '/' fName];`
`fPathAndName=[evalin('caller', 'pwd') '\' fName];`

also the line
`fNamestr=strfind(fCallerInformation.file,'/');`
can be replaced with
`fNamestr=strfind(fCallerInformation.file,filesep);`